src/browser/atom-application.coffee

```diff
@@ -167,7 +167,7 @@ class AtomApplication
     @on 'application:run-benchmarks', -> @runBenchmarks()
     @on 'application:quit', -> app.quit()
     @on 'application:new-window', -> @openPath(_.extend(windowDimensions: @focusedWindow()?.getDimensions(), getLoadSettings()))
-    @on 'application:new-file', -> (@focusedWindow() ? this).openPath()
+    @on 'application:new-file', -> @openPath()
```

Comment:
> If there is no focused window, then focusedWindow returns undefined and the command is ignored. Removing this check seemed like a quick fix.

I don't think your fix will have the desired effect. With the original code, if `@focusedWindow()` returned `undefined`, then `this.openPath()` would be called. Your change makes it so that `this.openPath()` is *always* called, even if there *is* a focused window.